setup.py

```diff
@@ -5,32 +5,26 @@
 
 setuptools.setup(
     name="nexus_keycode",
-    packages=setuptools.find_packages(),
+    packages=["nexus_keycode"],
     license="MIT",
-    version="1.0.2",
+    version="1.1.0",
     author="Angaza, Inc.",
     author_email="iot@angaza.com",
-    description="The Angaza Nexus Keycode for PAYG devices",
+    description="Angaza Nexus backend libraries for managing PAYG devices",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/angaza/nexus-keycode-python",
-    download_url="https://github.com/angaza/nexus-keycode-python/releases/download/1.0.2/nexus_keycode-1.0.2.tar.gz",
-    install_requires=[
-        "bitstring>=3.0.2",
-        "enum34==1.1.6",
```

Comment:
Do we still need this backport for `enum34` on Py2?

Reply:
Yeah, I guess you're right. It actually doesn't matter because `bitstring` (or one of its dependencies) also depends on `enum34`, so it gets pulled into a new installation anyway. But still better to call it out explicitly when we're using it directly.